### DISCUSSION TITLE
map `s` to `C-w` is good idea?

### DISCUSSION THREAD

**Discussion by cutbypham** [LazyVim/LazyVim#541] [General]
- in the daily use, i don't remember use `s` for something, `cl`  = `s`.
- in the past i map `sv` and `ss` for split window.
- so today i decide map `s` to `C-w`.

Pro: single key press, home row key.
Con: is not default mapping.

what are you think about this? is bad habit?

**Comment by loiccoyle** [comment]
FYI: `s` is mapped to [`leap`](https://github.com/ggandor/leap.nvim) by default

**Reply by cutbypham** [reply]
yup, i forgot, i disable leap, flit :smile: 
```lua
	{ "ggandor/leap.nvim", enabled = false },
	{ "ggandor/flit.nvim", enabled = false },
```

**Comment by Cruising0904** [comment]
as long as you don't use s it would be great I think. I personally use s more than c

**Comment by folke** [comment]
fyi, in LazyVim, `<c-s>` is also mapped to save the file, but if you don't use leap, `s` works too
